components/ButtonUrl/inde.js

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+export default function ButtonUrl() {
+  return <button>Add Recipe</button>;
+}
```

Comment:
It would be better to create only one button component you could use for different cases.